<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MoreVasanthKumar" failures="2" tests="2" name="com.runner.TestRunner" time="18.674" errors="0" timestamp="2023-11-07T18:02:04 IST" skipped="0">
  <testcase classname="com.runner.TestRunner" name="runScenario" time="6.335">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.StepDefinitions.LoginStepDef.user_enters_user_name_and_password(LoginStepDef.java:23)
at ✽.user enters userName and password(file:///C:/Users/SPUGE/git/cucumberE2E/CucumberProject/src/test/resources/FeatureFiles/LoginFeature.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="com.runner.TestRunner" name="runScenario" time="12.339">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.StepDefinitions.LoginStepDef.user_enters_user_name_and_password(LoginStepDef.java:23)
at ✽.user enters userName and password(file:///C:/Users/SPUGE/git/cucumberE2E/CucumberProject/src/test/resources/FeatureFiles/LoginFeature.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.runner.TestRunner -->
